multiqc/modules/vcftools/relatedness.py

```diff
@@ -0,0 +1,48 @@
+import csv
+from collections import defaultdict
+from multiqc.plots import heatmap
+
+
+class Relatedness2Mixin():
+    def parse_relatedness2(self):
+        for f in self.find_log_files('vcftools/relatedness2', filehandles=True):
+            m = _RelatednessMatrix(f)
+
+            self.add_section(
+                'Vcftools relatedness2 ' + m.name,
```

Comment:
Please used named arguments here if you can, in case I change stuff in the future...